web/src/survey-primary.json

```diff
@@ -9728,7 +9734,8 @@
      "visibleIf": "{surveyEnvironment} = 'DEV'"
     }
    ],
-   "visibleIf": "{surveyEnvironment} = 'DEV' or isContainerReady('applicantPage')",
+   "visible": false,
+   "visibleIf": "isContainerReady('applicantPage' or {surveyEnvironment} = 'DEV'",
```

Comment:
Malformed here.

Reply:
i fixed this one in editor1 and the page still didn't show up.